package.json

```diff
@@ -98,6 +98,7 @@
     "classnames": "^2.2.5",
     "dateformat": "^1.0.12",
     "deep-equal": "^1.0.1",
+    "fetch-mock": "^5.12.1",
```

Comment:
Shouldn't this be a dev-dependency since it is only used for testing?

Reply:
Yeah this should be updated.